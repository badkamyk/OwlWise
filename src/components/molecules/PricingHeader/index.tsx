import ContentHeading from "../../atoms/ContentHeading";
import MainContentParagraph from "../../atoms/MainContentParagraph";

export default function PricingHeader() {
	return (
		<div className="mx-auto max-w-screen-md text-center mb-8 lg:mb-12">
			<ContentHeading text={"Designed for business teams like yours"} size={"4xl"} color={"white"}/>
			<MainContentParagraph color={"white"} size={"lg"}
				text={"Here at OwlWise we focus on markets where technology, innovation, and capital can unlock long-term value and drive economic growth"} />
		</div>
	);
}