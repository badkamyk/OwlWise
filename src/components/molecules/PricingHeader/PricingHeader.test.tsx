import { render, screen } from "@testing-library/react";
import { expect, it } from "vitest";
import PricingHeader from "./index";

it("renders PricingHeader correctly", () => {
	const { container } = render(<PricingHeader />);
	expect(
		container.getElementsByClassName(
			"mx-auto mb-8 max-w-screen-md text-center lg:mb-12"
		)
	).toBeTruthy();
	expect(container.getElementsByClassName("text-4xl text-white")).toBeTruthy();
	expect(
		screen.getByText("Designed for business teams like yours")
	).toBeTruthy();
	expect(container.getElementsByClassName("text-lg text-white")).toBeTruthy();
	expect(
		screen.getByText(
			"Here at OwlWise we focus on markets where technology, innovation, and capital can unlock long-term value and drive economic growth"
		)
	).toBeTruthy();
	expect(container).toMatchSnapshot();
});
