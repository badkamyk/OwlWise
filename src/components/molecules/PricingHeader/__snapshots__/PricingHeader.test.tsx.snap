// Vitest Snapshot v1

exports[`PricingHeader 1`] = `
<div>
  <div
    class="mx-auto mb-8 max-w-screen-md text-center lg:mb-12"
  >
    <h3
      class="mb-4 tracking-tight font-extrabold text-white text-4xl"
    >
      Designed for business teams like yours
    </h3>
    <p
      class="mb-4 text-white text-lg"
    >
      Here at OwlWise we focus on markets where technology, innovation, and capital can unlock long-term value and drive economic growth
    </p>
  </div>
</div>
`;
